RHDHBUGS-2864 - Tekton plugin bundled in Red Hat Developer Hub 1.9 does not work (#2148)
redhat-developer/red-hat-developers-documentation-rhdh: modules/shared/proc-enable-the-tekton-plugin.adoc

When you use the label selector, the mentioned labels must be present on the resource.

.Procedure

. To enable the Tekton plugin, you must update the plugin settings in your application configuration. 

> .. If you are using the Tekton plugin that is pre-loaded in {product-very-short}, in addition to setting the `disabled` property to `false`, you must also add the plugin configuration as follows:

Reviewer: ```suggestion
.. If you are using the Tekton plugin that is pre-loaded in {product-very-short}, you must set the `disabled` property to `false`, and modify the plugin configuration as follows:
```
